packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -64,6 +64,7 @@ export interface IOpsCachingPolicy {
  */
 export interface ICollabSessionOptions {
 	/**
+	 * @deprecated No longer needed
```

Comment:
Could you please put (in comment, human readable - like RC3) a version when it's deprecated - simplifies the work to delete it in the future.